### ISSUE TITLE
10spd segementation anchors

### ISSUE THREAD

**Issue by tischi** [constantinpape/autophagosomes-clem#8]
@constantinpape 
The 10spd segementation anchors do not seem correct.
Maybe they are still in pixel units?
The coordinate have a very different magnitude compared with the coordinates in the image:
![image](https://user-images.githubusercontent.com/2157566/83305277-bfcffc80-a200-11ea-85f0-d536d6dc73fb.png)

**Comment by constantinpape** [comment]
No, I think the issue is that the segmentation doesn't make sense, because it just has two objects.
I added this one only to test writing the tables, I will update it with a meaningful segmentation eventually. We can leave this open until then.
